agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForTelemtryPlatform.cs

```diff
@@ -10,63 +10,63 @@
 namespace Agrirouter.Api.Test.Service.Onboard
 {
     /// <summary>
-    /// Functional tests.
+    ///     Functional tests.
     /// </summary>
     public class SecuredOnboardingServiceTestForTelemtryPlatform : AbstractSecuredIntegrationTestForTelemetryPlatform
     {
         private static readonly HttpClient HttpClient = HttpClientFactory.HttpClient();
 
-        [Fact(Skip = "Will not run successfully without changing the registration code.")]
-        public void GivenValidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
+        [Fact]
+        public void GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
         {
             var onboardingService =
                 new SecuredOnboardingService(Environment, HttpClient);
 
             var parameters = new OnboardParameters
             {
-                Uuid = Guid.NewGuid().ToString(),
+                Uuid = GetType().FullName,
```

Comment:
Why aren't you using Guid.NewGuid().ToString() here? The test is about an invalid Registration code but you add an additional potential issue?